atlas-engine/ProcessEngine.CompatibilityProxyAdapter
# ✨ Add Proxy Adapter for the Management API

## Changes

Implement the Proxy Adapter that maps Management API requests against the AtlasEngine.

## Issues

PR: #4

## How to test the changes

- Start the Fullstack Server 
- Use the Management API client to make requests against the Management API endpoints
- See that it works
